lucien-ios/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LoginManager.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/29/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+final class LoginViewModel {
+
+    let lucienAPIClient = LucienAPIClient()
+
+    func getCurrentUser() {
+        lucienAPIClient.getCurrentUser { response in
+            switch response {
+            case .success(let user):
+                print(user)
```

Comment:
These prints are fine for now since the API isn't hooked up, but in the future leave a TODO saying they'll be removed